1904-11-04.xml

```diff
@@ -0,0 +1,2581 @@
+                    <p><date when="1904-11-07">FRIDAY, NOVEMBER 4, 1904.</date>.</p>
+                </div>
+                <div type="item">
+                    <head>ANGLO-FRENCH FRIENDSHIP</head>
+                    <p>Yesterday the French Parliament entered on the discussion of the report of M.
+                        Deloncle on the Anglo-French Convention. It was pure- ly an accident that
+                        there has been delay in proceeding to ratification. On our side of the
+                        Channel that stage did not receive Parlia- mentary accomplishment until the
+                        very end of last Session, while in France time for doing so could not be
+                        either found or made, despite M. Delcassé's strenuous endeavours to hurry up
+                        the Chamber. Happily, there was-as there deserved to be-such a strong
+                        conviction of good faith and sincerity on both sides that the outside world
+                        came to regard the Convention as in actual being and opera- </p>
+                </div>
+                <cb n="6"/>
+                <!-- Sunlight soap untemplated -->
+                <div type="item">
+                    <p>tion. Neither in the United Kingdom nor elsewhere was there ever the faintest
+                        doubt that when the Chamber re-assembled it would take the matter in hand
+                        with all possible dis- patch. That expectation is now fulfilled, and too
+                        great credit could not be given to M. Cambon and M. Delcassé for the
+                        patience and perseverance with which they helped Lord Lansdowne to smooth
+                        away outstand- ing difficulties. But they could have done little or nothing
+                        had not French feeling at large expressed itself unmistakably in favour of
+                        strengthening and extending the entente between the neighbouring nations.
+                        The French Colonial party having bestowed its blessing on. the concordat
+                        diplomacy had an easy labour in giving the finishing touches to the New-
+                        foundland Agreement. Here, again, it is only fair to admit that M. Deloncle
+                        has full right to claim the renunciation by France of an irritat- ing clause
+                        of the Treaty of Utrecht as an "important concession." It is of that
+                        character, unquestionably, and although an equivalent has been given by
+                        Great Britain elsewhere, the surrender by France of such very ancient rights
+                        and privileges denotes genuine goodwill towards England. </p>
+                </div>
+                <div type="section" feature="wire">
+                    <head>TO-DAY'S TELEGRAMS</head>
+                    <div type="item" feature="wire">
```

Comment:
If the parent section has a feature attribute, you don't need to add the same attribute to child items inside the section.